web/client/src/library/components/fileTree/Directory.tsx

```diff
@@ -97,7 +97,7 @@ export default function Directory({
 
     setIsLoading(true)
 
-    const extension = '.py'
+    const extension = directory.isModels ? '.sql' : '.py'
```

Comment:
What does the user need to do to create a Python model?

Reply:
Create default file and rename to .py